spec/unit/unparser_spec.rb

```diff
@@ -267,6 +267,16 @@ def foo(bar:, baz: "value")
           assert_source '{ :-@ => 1 }'
         end
       end
+
+      context 'method' do
```

Comment:
I did not tried. But I think this example can be reduced to a smaller core. At least it should get a more descriptive explanation than `method`.